mmcv/utils/path.py

```diff
@@ -60,19 +60,13 @@ def scandir(dir_path, suffix=None, recursive=False):
     root = dir_path
 
     def _scandir(dir_path, suffix, recursive):
-        for entry in os.scandir(dir_path):
-            if not entry.name.startswith('.') and entry.is_file():
-                rel_path = osp.relpath(entry.path, root)
-                if suffix is None:
-                    yield rel_path
-                elif rel_path.endswith(suffix):
-                    yield rel_path
-            else:
-                if recursive:
-                    yield from _scandir(
-                        entry.path, suffix=suffix, recursive=recursive)
-                else:
-                    continue
+    for entry in os.scandir(dir_path):
```

Comment:
CI raises IndentationError

Reply:
I believe I fixed that. I copied/pasted proposed change but it had weird indentation.